win32ss/gdi/ntgdi/cliprgn.c

```diff
@@ -804,6 +811,13 @@ CLIPPING_UpdateGCRegion(PDC pDC)
         }
     }
 
+    if (!pDC->prgnRao)
```

Comment:
Why don't you move this warning (+ return) just after where the prgnRao is created? (i.e. line 785)
Or maybe, instead, move the prgnRao allocation down here?

Reply:
To give the code above a chance to run.

Reply:
ok then what about moving the rao alloc down too? :thinking:

Reply:
Sounds good, will do.